.github/workflows/csharp.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       TRAVIS_COM_TOKEN: ${{ secrets.TRAVIS_COM_TOKEN }}
   
   fullstack_production_suite:
-    name: Run Full Stack Compatibility Suite
+    name: Run Optimizely Feature Experimentation Compatibility Suite
```

Comment:
Totally fine with the change so long as you don't think it would be confusing since we still call the test suite by "FSC".

Reply:
I am not sure are we are going to rename FSC as well?

Reply:
Funny. What can be a good name to replace it internally. FXC? @russell-loube-optimizely